cfpb/design-system-react
# Update Storybook UI theme to improve readability

Closes #136 
Closes #156 
Closes #56 

## Changes
  - Use `Avenir Next` as default font for all Storybook text
  - Sentence case for story names
  - Allow user to switch between light/dark Storybook theme
  - CFPB logo for branding

## How to test this PR

1. Check out the screenshots below
2. `yarn && yarn start` -> Poke around, see how things feel: are they readable? 

## Screenshots
|Light theme|Dark theme|
|---|---|
|<img width="894" alt="Screenshot 2023-09-05 at 11 40 28 AM" src="https://github.com/cfpb/design-system-react/assets/2592907/79b6ba33-c449-402e-a2c8-b664c9e477c9">|<img width="895" alt="Screenshot 2023-09-05 at 11 40 15 AM" src="https://github.com/cfpb/design-system-react/assets/2592907/eaca790b-0c7d-4605-a925-33b64dec8689">|
